multi_block_core.dic

```diff
@@ -120,14 +120,14 @@ save_CELL
 ;
     _name.category_id             DIFFRN
     _name.object_id               CELL
-    _category_key.name            '_cell.diffrn_id'
+    _category_key.name            '_cell.structure_id'
 
 save_
 
 save_cell.diffrn_id
 
     _definition.id                '_cell.diffrn_id'
-    _definition.update            2023-02-01
+    _definition.update            2023-10-26
     _description.text
 ;
     A pointer to the diffraction conditions to which this cell has been applied,
```

Comment:
`_cell.diffrn_id` is unnecessary as it has the same value as `_structure.diffrn_id`.